app/src/test/java/org/oppia/android/app/player/state/StateFragmentLocalTest.kt

```diff
@@ -282,7 +283,37 @@ class StateFragmentLocalTest {
   }
 
   @Test
-  fun testStateFragment_nextState_submitCorrectAnswer_confettiIsDisplayed() {
+  @Config(qualifiers = "land")
+  fun testStateFragment_nextState_submitCorrectAnswer_correctTextBannerIsDisplayedInLandscape() {
+    launchForExploration(FRACTIONS_EXPLORATION_ID_1).use {
+      startPlayingExploration()
+      playThroughState1()
+
+      // Submit correct answer
+      submitFractionAnswer(answerText = "3/4")
+
+      onView(withId(R.id.congratulations_text_view))
+        .check(matches(isCompletelyDisplayed()))
+    }
+  }
+
+  @Test
+  @Config(qualifiers = "port")
+  fun testStateFragment_nextState_submitCorrectAnswer_confettiIsDisplayedInPortrait() {
```

Comment:
Here & elsewhere in this PR: it may be clearer to keep landscape/portrait as a condition rather than wording it as part of the outcome, e.g.:

``testStateFragment_portrait_nextState_submitCorrectAnswer_confettiIsDisplayed``

Reply:
Done